dev/dll/XamlControlsResources.cpp

```diff
@@ -204,47 +204,61 @@ void XamlControlsResources::UpdateAcrylicBrushesDarkTheme(const winrt::IInspecta
+            UpdateTintLuminosityOpacity(acrylicInAppFillColorDefaultInverseBrush, 0.85);
         }
         if (const auto acrylicBackgroundFillColorBaseBrush = dictionary.Lookup(box_value(c_AcrylicBackgroundFillColorBaseBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            acrylicBackgroundFillColorBaseBrush.TintLuminosityOpacity(0.96);
+            UpdateTintLuminosityOpacity(acrylicBackgroundFillColorBaseBrush, 0.96);
         }
         if (const auto acrylicInAppFillColorBaseBrush = dictionary.Lookup(box_value(c_AcrylicInAppFillColorBaseBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            acrylicInAppFillColorBaseBrush.TintLuminosityOpacity(0.96);
+            UpdateTintLuminosityOpacity(acrylicInAppFillColorBaseBrush, 0.96);
         }
         if (const auto accentAcrylicBackgroundFillColorDefaultBrush = dictionary.Lookup(box_value(c_AccentAcrylicBackgroundFillColorDefaultBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            accentAcrylicBackgroundFillColorDefaultBrush.TintLuminosityOpacity(0.8);
+            UpdateTintLuminosityOpacity(accentAcrylicBackgroundFillColorDefaultBrush, 0.8);
         }
         if (const auto accentAcrylicInAppFillColorDefaultBrush = dictionary.Lookup(box_value(c_AccentAcrylicInAppFillColorDefaultBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            accentAcrylicInAppFillColorDefaultBrush.TintLuminosityOpacity(0.8);
+            UpdateTintLuminosityOpacity(accentAcrylicInAppFillColorDefaultBrush, 0.8);
         }
         if (const auto accentAcrylicBackgroundFillColorBaseBrush = dictionary.Lookup(box_value(c_AccentAcrylicBackgroundFillColorBaseBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            accentAcrylicBackgroundFillColorBaseBrush.TintLuminosityOpacity(0.8);
+            UpdateTintLuminosityOpacity(accentAcrylicBackgroundFillColorBaseBrush, 0.8);
         }
         if (const auto accentAcrylicInAppFillColorBaseBrush = dictionary.Lookup(box_value(c_AccentAcrylicInAppFillColorBaseBrush)).as<winrt::Windows::UI::Xaml::Media::AcrylicBrush>())
         {
-            accentAcrylicInAppFillColorBaseBrush.TintLuminosityOpacity(0.8);
+            UpdateTintLuminosityOpacity(accentAcrylicInAppFillColorBaseBrush, 0.8);
         }
     }
 }
 
+void XamlControlsResources::UpdateTintLuminosityOpacity(winrt::Windows::UI::Xaml::Media::AcrylicBrush brush, double luminosityValue)
+{
+    if (SharedHelpers::Is20H1OrHigher())
+    {
+        brush.TintLuminosityOpacity(luminosityValue);
+    }
+    else
+    {
+        // On 19h1 and 19h2 there is a heap corruption bug that occurs when setting the TintLuminosityOpacity via the property,
+        // So we'll set it by the DP instead.
+        brush.SetValue(winrt::Windows::UI::Xaml::Media::AcrylicBrush::TintLuminosityOpacityProperty(), box_value(luminosityValue));
```

Comment:
IIRC the only difference is that this doesn't trigger the "notify on property change" handlers, which is probably ok for this prop but would be worth adding a comment for